src/RadioGroup.js

```diff
@@ -12,7 +12,7 @@ class RadioGroup extends React.Component {
         }),
         container: PropTypes.string,
         name: PropTypes.string.isRequired,
-        onChange: PropTypes.func.isRequired,
+        onChange: PropTypes.func,
```

Comment:
Exactly my reasoning. React already handles the checking and it's not required on the true input element. Warnings will show as they should, and the user will interact with the element like they would a normal input element.

As a user, I might not care to listen to the `onChange` event. I may be prototyping.

Especially in the case of the other input types, I often use `defaultValue` instead of `value`+`onChange`

Making the fewest assumptions about what the user "meant to do" keeps the library from limiting what they can do